### PR TITLE
Publish preparation

### PR DESCRIPTION
**Pull request by marcos-iov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
+<img src="./rootstock_logo.png" alt="Rootstock" />
+
 # rsk-transaction-helper
+![Github CI/CD](https://github.com/rsksmart/rsk-transaction-helper/actions/workflows/workflow.yml/badge.svg)
+[![CodeQL](https://github.com/rsksmart/rsk-transaction-helper/workflows/CodeQL/badge.svg)](https://github.com/rsksmart/rsk-transaction-helper/actions?query=workflow%3ACodeQL)
```
[the remark this answers is not in the document]
Comment (marcos-iov) [reply README.md]: That's ok there, see where it closes

File: package.json [hunk 2]
```diff
@@ -1,15 +1,31 @@
 {
-  "name": "rsk-transaction-helper",
-  "version": "3.2.0",
-  "description": "Utilitary library to send transactions on RSK using Web3 and Ethereum-js",
+  "name": "rootstock-transaction-helper",
```
Comment (kelvinator07) [inline package.json:2]:
```suggestion
  "name": "@rsksmart/rootstock-transaction-helper",
```

File: package.json [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
     "sample": "node ./samples/send-transaction.js"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rsksmart/rsk-transaction-helper.git"
+  },
+  "bugs": {
+    "url": "https://github.com/rsksmart/rsk-transaction-helper/issues"
+  },
+  "homepage": "https://github.com/rsksmart/rsk-transaction-helper#readme",
```
Comment (kelvinator07) [inline package.json:18]: The urls Should Be *rootstock-transaction-helper

Comment (marcos-iov) [reply package.json]: Good catch. Thanks!

File: README.md [hunk 4]
```diff
@@ -1,4 +1,9 @@
-# rsk-transaction-helper
+<img src="./rootstock_logo.png" alt="Rootstock" />
+
+# rootstock-transaction-helper
+![Github CI/CD](https://github.com/rsksmart/rootstock-transaction-helper/actions/workflows/workflow.yml/badge.svg)
+[![CodeQL](https://github.com/rsksmart/rootstock-transaction-helper/workflows/CodeQL/badge.svg)](https://github.com/rsksmart/rootstock-transaction-helper/actions?query=workflow%3ACodeQL)
+
 Utility library to send transactions on RSK using Web3 and Ethereum-js
```
Comment (jeremy-then) [inline README.md:7]: Should this be `Utility library to send transactions on Rootstock using Web3 and Ethereum-js`?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nathanieliov) [reply README.md]: Oh cool.

**Review by nathanieliov — APPROVED** [review]

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**